core/templates/dev/head/pages/exploration-editor-page/history-tab/history-tab.directive.spec.ts

```diff
@@ -102,6 +102,7 @@ describe('HistoryTab controller', function() {
     beforeEach(angular.mock.inject(function(_$componentController_) {
       $componentController = _$componentController_;
       historyTabCtrl = $componentController('historyTab', null, {});
+      historyTabCtrl.$onInit();
```

Comment:
Explanation for this change: https://www.codelord.net/2017/01/09/unit-testing-angular-components-with-%24componentcontroller/